x-pack/plugin/core/src/main/resources/fleet-file-data.json

```diff
@@ -26,6 +26,10 @@
             "type": "keyword",
             "index": false
```

Comment:
we probably need to enable indexing for this field, we are querying based on it in fleet server